test/sm2_internal_test.c

```diff
@@ -251,9 +252,35 @@ static int sm2_crypt_test(void)
             "88E3C5AAFC0413229E6C9AEE2BB92CAD649FE2C035689785DA33"))
         goto done;
 
+    /* From GM/T0003.5-2012, Appendix A part.*/
+    gm_group = create_EC_group
+        ("fffffffeffffffffffffffffffffffffffffffff00000000ffffffffffffffff",
```

Comment:
Nit: the opening parenthesis should be on the previous line, by the function name